Stellar-transaction.x

```diff
@@ -578,6 +579,22 @@ struct InvokeHostFunctionOp
     SorobanAuthorizationEntry auth<>;
 };
 
+enum BumpExpirationType
+{
+    BUMP_EXPIRATION_UNIFORM = 0
+};
+
+/* Bump the expiration ledger of the entries specified in the readOnly footprint
```

Comment:
Is doing a bump via RO entries more efficient than via RW entries? I suppose it is (because we should write the expiration entries only), but just wanted to double-check.

Reply:
Yeah, RO should always be more efficient

Reply:
Yeah it would also be inaccurate since we wouldn't be writing those entries (I should note though that the current implementation doesn't use EXTENSION_ENTRIES, but that change will be made soon).